doc/ext/imgconverter.rst

```diff
@@ -0,0 +1,30 @@
+.. highlight:: rest
+
+:mod:`sphinx.ext.imgconverter` -- Convert images to appropriate format for builders
```

Comment:
I need to advice the module name. Is it better to use `imagemagick` for its name?